lib/constructs/interface.js

```diff
@@ -1120,14 +1120,22 @@ class Interface {
     }
 
     this.str += `
-      create(constructorArgs, privateData) {
-        let obj = Object.create(${this.name}.prototype);
-        obj = this.setup(obj, constructorArgs, privateData);
+      create(globalObject, constructorArgs, privateData) {
+        if (globalObject[ctorRegistry] === undefined) {
+          throw new TypeError('Invalid global object');
```

Comment:
```suggestion
          throw new Error('Internal error: invalid global object');
```

Reply:
I think these errors should never come up unless we mess up as jsdom maintainers. Right? So let's prefix them with "Internal error".